network-api/networkapi/mozfest/templates/fragments/blocks/mozfest_stats_block.html

```diff
@@ -0,0 +1,16 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+    <div class="tw-bg-black tw-dark tw-py-16">
+        <div class="tw-container">
```

Comment:
Do `tw-container` and the other wrapper need to be separate?

Reply:
By 'the other wrapper' do you mean `tw-row`? They need to be separate as they both set different left and right margin values - `tw-container` sets them as `auto` to center align the content and `tw-row` sets them a `-1rem` for the grid system in use.

Reply:
I meant the "wrapper" with `<div class="tw-bg-black tw-dark tw-py-16">`. There does not seem to clashing definitions between those two.

Reply:
Ah i'm with you. 

Combining them would mean that the black background wouldn't stretch across the screen due to the `max-width` on `tw-container`:

![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/31627284/26d65376-a8fb-4ce7-adc1-a29b6573c18d)

Reply:
Aaah 💡 that makes sense. I did not know about the containers width restriction. Thanks for this. 

Also like the fitting example text 😅